### PR TITLE
A few small changes:

### PR DESCRIPTION
**Pull request by andrewrlee**
### Context

> Does this issue have a Trello card?

No, this just covers a few documentation fixes and a logging fix

> If this is an issue, do we have steps to reproduce?

### Intent

> What changes are introduced by this PR that correspond to the above card?

 * Removing unused env vars and incorrect endpoint url from env template
 * Updating doc'd node version
 * Only log missing hot jar at startup rather than every request

> Would this PR benefit from screenshots?

No

### Considerations

> Is there any additional information that would help when reviewing this PR?

No 

> Are there any steps required when merging/deploying this PR?

### Checklist

- [ ] This PR contains **only** changes related to the above card
- [ ] Tests have been added/updated to cover the change
- [ ] Documentation has been updated where appropriate
- [ ] Tested in Development

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -224,12 +224,12 @@ const createApp = ({
   }
 
   // Routing
+  if (!process.env.HOTJAR_ID) {
```
Comment (andrewrlee) [inline server/app.js:227]: this was a bit noisy - so moved to just on start up

Comment (spikeheap) [reply server/app.js:227]: 🙌 would it be worth sending this warning to Sentry as well?

**Review by spikeheap — DISMISSED** [review]

**Review by davidthomas22 — DISMISSED** [review]
LGTM

File: server/app.js (hunk 1 above)
Comment (andrewrlee) [reply server/app.js:227]: added capturing a message in sentry. 

Docs say that these are not usually emitted. not sure if it's specific config we need to enable

**Review by davidthomas22 — APPROVED** [review]
LGTM

**Pull request merged**